fastlane/lib/fastlane/actions/update_project_provisioning.rb

```diff
@@ -130,7 +138,11 @@ def self.available_options
           FastlaneCore::ConfigItem.new(key: :certificate,
                                        env_name: "FL_PROJECT_PROVISIONING_CERTIFICATE_PATH",
                                        description: "Path to apple root certificate",
-                                       default_value: "/tmp/AppleIncRootCertificate.cer")
+                                       default_value: "/tmp/AppleIncRootCertificate.cer"),
+          FastlaneCore::ConfigItem.new(key: :codesigning_identity,
```

Comment:
Can we also change this key to `code_signing_identity`?

Reply:
Thanks for ur advice I had update the code